src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -225,5 +227,206 @@ public static String getAttributeTableName(String clsName,
+			}
+		}
+		return eventsNotUsedForInference;
+	}
+
+	/**
+	 * Checks if incoming Event is manually inferred by checking inferredFrom referral.
+	 * This method only correctly identifies manual inferences in a curation or slice database.
+	 * This is due to the fact that it only checks the 'inferredFrom' attribute, which is used
+	 * in automatic inferences as well.
+	 * @param event GKInstance -- Event instance being checked for inferredFrom referral.
+	 * @return boolean -- true if inferredFrom referral exists, false if not.
+	 * @throws Exception -- Thrown by MySQLAdaptor.
+	 */
+	private static boolean manuallyInferred(GKInstance event) throws Exception {
+		return event.getReferers(ReactomeJavaConstants.inferredFrom) != null;
+	}
+
+	/**
+	 * Finds all Events not used for Inference, and then finds subset that are Human ReactionlikeEvents
+	 * @param dba MySQLAdaptor
+	 * @param skiplistDbIds List<String> -- List of Pathway DbIds. If an Event being checked is a member of these pathways, they are skipped.
+	 * @return Set<GKInstance> -- All Human ReactionlikeEvents that are not used for manual inference.
+	 * @throws Exception-- Thrown by MySQLAdaptor
+	 */
+	public static Set<GKInstance> findHumanReactionsNotUsedForManualInference(MySQLAdaptor dba, List<String> skiplistDbIds) throws Exception {
+		Set<GKInstance> reactionsNotUsedForManualInference = new HashSet<>();
+		for (GKInstance event : findEventsNotUsedForManualInference(dba, skiplistDbIds)) {
+			// Filter for Human ReactionlikeEvents
+			if (event.getSchemClass().isa(ReactomeJavaConstants.ReactionlikeEvent)
+					&& isHumanDatabaseObject(event)) {
+
+				reactionsNotUsedForManualInference.add(event);
+			}
+		}
+		return reactionsNotUsedForManualInference;
+	}
+
+	/**
+	 * This method utilizes the InstanceUtilities method for finding all reaction participants. It also checks for the
+	 * 'activeUnit' participant in catalysts/regulations that can be flagged by certain QA checks. Generally speaking,
+	 * the 'activeUnit' instance should be found in the 'physicalEntity' of catalysts/regulations. Sometimes this is not
+	 * the case, when the wrong activeUnit is mistakenly checked in. This will be flagged by QA tests that call this method.
+	 * @param reaction GKInstance -- ReactionlikeEvent that will be checked for all participants.
+	 * @return Set<GKInstance> -- All distinct participants in the incoming ReactionlikeEvent.
+	 * @throws Exception -- Thrown by MySQLAdaptor.
+	 */
+	public static Set<GKInstance> getAllReactionParticipantsIncludingActiveUnits(GKInstance reaction) throws Exception {
+		Set<GKInstance> reactionPEs = new HashSet<>();
+		reactionPEs.addAll(InstanceUtilities.getReactionParticipants(reaction));
+		// Catalysts/Regulations also added to check for incorrect activeUnits.
+		reactionPEs.addAll(reaction.getAttributeValuesList(ReactomeJavaConstants.catalystActivity));
+		reactionPEs.addAll(reaction.getAttributeValuesList(ReactomeJavaConstants.regulatedBy));
```

Comment:
So, my understanding is the lines getting the catalyst activities and regulations are so they can be investigated to grab the physical entities in the `activeUnit` attribute not grabbed by the call to `InstanceUtilities.getReactionParticipants`?

If this is right, I find it a bit confusing to add the catalyst activity and regulation instances directly to the `reactionPEs` set since they are not Physical Entity instances themselves but contain physical entities.  Would it be possible to hold them in a different set that is sent a method responsible only for querying activeUnit for physical entities and returning them to be added to the `reactionPEs`?  Then you could send the elements of `reactionPEs` to be recursed to get physical entities contained physical entities.